integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -160,6 +160,16 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
     }
   }
 
+  private void waitForFilePersisted(long fileId) {
```

Comment:
I am concerned with the way we are using Whitebox in here. It really breaks the encapsulation and requires a lot internal knowledge, meanwhile easy to break once we change the class to be tested.

Can we make a `MockFileSystemWorker` to have this method?